assets/src/edit-story/elements/media/edit.js

```diff
@@ -107,25 +109,62 @@ function MediaEdit({ element, box }) {
   const [croppedMedia, setCroppedMedia] = useState(null);
   const [cropBox, setCropBox] = useState(null);
   const elementRef = useRef();
+  const scaleRef = useRef(null);
```

Comment:
Default null is required here to make `useKeyDownEffect` work correctly when the element hasn't mounted yet.